features/support/env.rb

```diff
@@ -41,6 +41,16 @@
 end
 
 
+Before do
+  @error_expected = false
+end
+
+
+After do
+  expect(@last_run_result.error).to be_falsy unless @error_expected
```

Comment:
We should check both cases here:
* that there was an error if we expected one
* that there was no error if we didn't expect one